kong/templates/kong_defaults.lua

```diff
@@ -39,6 +39,9 @@ client_body_buffer_size = 8k
 error_default_type = text/plain
 
 nginx_http_ssl_protocols = TLSv1.1 TLSv1.2 TLSv1.3
+nginx_http_ssl_prefer_server_ciphers = on
+nginx_http_ssl_session_tickets = off
```

Comment:
Actually let's keep the `session_tickets` as-is for now. I know it has a security tradeoff but disabling it could have huge impact on session resumption rate and cause performance regressions, and better leave this decision to the user.

Reply:
@dndx it is not a small security tradeoff, it basically destroys perfect forward secrecy:
https://github.com/mozilla/server-side-tls/issues/135

Reply:
@dndx what I mean is that you can only fix it with:
`ssl_session_ticket_key` + `reloading` OR `restarting` Kong daily.

Reply:
No, it doesn't necessarily destroys PFS, it only destroys PFS if Nginx has not be restarted for months or years **and** the memory of Nginx is compromised, which is relatively unlikely to happen. But if we  do decide to disable it I would like to see some big warning coming in the release notes as there will likely be a performance regression.

Reply:
ok, I changed the defaults and left only `protocols`, `ciphers` and `prefer_server_ciphers` on the `suite` configuration.